tests/unit_test/app_common/filters/exclude_vars_test.py

```diff
@@ -19,15 +19,15 @@
 from nvflare.app_common.filters import ExcludeVars
 
 TEST_CASES = [
-    [{"a": 1.0, "b": 2.0}, "a", {"b": 2.0}],
-    [{"a": 1.0, "b": 2.0, "c": 3.0}, ["a", "b"], {"c": 3.0}],
-    [{"a": 1.0, "b": 2.0, "c": 3.0, "d": 4.0}, ["a", "d"], {"b": 2.0, "c": 3.0}],
-    [
+    ({"a": 1.0, "b": 2.0}, "a", {"b": 2.0}),
```

Comment:
what does it mean for changes dict to tuple ?

Reply:
Logically it is the same.

I did this because the pycharm IDE is complaining.